src/main/java/com/poiji/config/Casting.java

```diff
@@ -10,4 +10,8 @@
 public interface Casting {
 
     Object castValue(Class<?> fieldType, String value, PoijiOptions options);
+
+    default Object castValue(Class<?> fieldType, String value, int row, int column, PoijiOptions options) {
```

Comment:
not sure why we need a "default" method here

Reply:
Went with a default method to ensure backwards compatibility with existing `Casting` implementations. Otherwise `DefaultCasting` would need to be modified, as would any user implementations out in the wild. 

Happy to make it a breaking change by removing the default implementation and modifying `DefaultCasting` if that's preferred.